### DISCUSSION TITLE
Tilemap service

### DISCUSSION THREAD

**Discussion by HemboPagi** [openseadragon/openseadragon#2126] [Q&A]
Hello

I have spent too many hours with that without the result so i will turn toward the community to get some help.

I have image tile set generated by the external service. json output is something like that:

"pages": [{
	"attributes": {
		"id": 76074,
		"width": 1701,
		"height": 758,
		"itemId": 4208,
		"tile": {
			"width": 512,
			"height": 512
		},
		"resolution": 0,
		"levels": 5,
		"label": 1
	},
	"tilemap": {
		"uri": "https:\/\/demo.mediainfo.com\/api\/page\/76074\/tile\/{level}\/{row}\/{column}",
		"level": []
	}
}

So the tiles are as tilemap .

Following the example from https://openseadragon.github.io/examples/tilesource-tms/ i managed to get that far that i have some blurry images on my screen and many wrong image request in my console log.
![Screenshot 2022-03-07 at 11 31 27](https://user-images.githubusercontent.com/95413906/157004751-1f0a0f4b-6a3e-47ba-af77-5d084e285e17.png)

Any ideas? When i turned on OSD debug i can see that the value for Level does not change during the zoom. Still, images what OSD request have level (the first) value changed.

![Screenshot 2022-03-07 at 11 38 23](https://user-images.githubusercontent.com/95413906/157005796-340254b3-7087-49a0-b7cb-a42752d90af0.png)

**Comment by iangilman** [comment]
Just noting there's further discussion on this here: https://github.com/openseadragon/openseadragon/issues/1365#issuecomment-1063419621
